cookbook/wallets.ts

```diff
@@ -0,0 +1,144 @@
+import path from "path"; // md-ignore
+import { KeyPair, Mnemonic, UserPem, UserWallet } from "../src"; // md-ignore
+// md-start
+(async () => {
+    // ## Wallets
```

Comment:
Maybe we should add a paragraph that directs users to the signing providers and / or sdk-dapp in the context of dApps. For example:

> In the context of dApp frontends, where the secret key of the user is not available (since it's securely enclosed in wallets), developers should rely on (...) in order to sign transactions or messages. On this topic, please see (...).